charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-network-policy.yaml

```diff
@@ -3,20 +3,22 @@ kind: NetworkPolicy
 metadata:
   annotations:
     gardener.cloud/description: |
-      Allows Egress from fluentbit to fluentd
+      Allows Egress from fluentbit to loki
   name: allow-fluentbit
   namespace: {{ .Release.Namespace }}
 spec:
   podSelector:
     matchLabels:
-{{ toYaml .Values.fluentbit.labels | indent 6 }}
+{{ toYaml .Values.labels | indent 6 }}
   egress:
   - to:
+     # Match all namespaces
+    - namespaceSelector: {}
```

Comment:
I think that this should be 

```yaml
- to:
  - namespaceSelector: {}
    podSelector: ....
```

Reply:
In fact it should be:

```yaml
spec:
  egress:
  - to:
    - namespaceSelector:
        matchLabels:
          gardener.cloud/role: shoot
      podSelector: 
{{ toYaml .Values.global.lokiLabels | indent 10 }}
    - podSelector: 
{{ toYaml .Values.global.lokiLabels | indent 10 }}
```

which means allow traffic to loki pods in shoot namespaces and in the current namespace to loki pods

Reply:
fixed